public/static/locales/cy/common.json

```diff
@@ -88,6 +88,7 @@
   "The establishment is the location or site of your food business. If it is a mobile food business, please use the location where it is normally stored overnight.": "Y sefydliad yw lleoliad neu safle eich busnes bwyd. Os yw'n fusnes bwyd symudol, defnyddiwch y lleoliad lle mae'n cael ei storio dros nos fel rheol.",
   "Registering Local Authority": "Awdurdod Lleol sy’n cofrestru",
   "is the local authority your registration will be sent to. Is this the correct local authority for your business? If unsure please use this [Food Business Registration](www.gov.uk/food-business-registration) link to check using the location or site of your food business.": "yw'r awdurdod lleol y bydd eich cofrestriad yn cael ei anfon ato. Ai hwn yw'r awdurdod lleol cywir ar gyfer eich busnes? Os ydych chi'n ansicr, defnyddiwch y ddolen [Cofrestru Busnesau Bwyd](https://www.gov.uk/food-business-registration) i wirio gan ddefnyddio lleoliad neu safle eich busnes bwyd.",
+  "is the local authority your registration will be sent to. Is this the correct local authority for your business? If unsure please use this [Food Business Registration link where you can check for your correct local authority](www.gov.uk/food-business-registration) using the location or site of your food business.": "yw'r awdurdod lleol y bydd eich cofrestriad yn cael ei anfon ato. Ai hwn yw'r awdurdod lleol cywir ar gyfer eich busnes? Os ydych chi'n ansicr, defnyddiwch y [ddolen Cofrestru Busnes Bwyd lle gallwch wirio am eich awdurdod lleol cywir](https://www.gov.uk/food-business-registration) gan ddefnyddio lleoliad neu safle eich busnes bwyd.",
```

Comment:
Why is this added instead of updating the old one above?

Reply:
removed the initial one now